tsconfig.json

```diff
@@ -0,0 +1,27 @@
+{
+  "compilerOptions": {
+    "baseUrl": "src",
+    "target": "ES2020",
```

Comment:
This seems to pass the tests so great, I had some issues with `jest` in another project that required this to change to `ES2019` so lets just keep an eye on it.